website/views.py

```diff
@@ -87,6 +91,12 @@ def messages(request):
     # Send them to the browser:
     return HttpResponse(json.dumps(chat_messages), content_type="application/json")
 
+def kill_messages(request):
+	# print(ChatMessage.objects)
+	for cm in ChatMessage.objects.filter():
```

Comment:
You can do this in a single query with `ChatMessage.objects.all().delete()`, which I would prefer because we have VERY DEMANDING performance needs on gnarchat.

Reply:
I'm sad this PR has sat without any attention for so long... :)